### DISCUSSION TITLE
Is it possible to find and replace text in multiple files simultaneously?

### DISCUSSION THREAD

**Discussion by StuporMundi1337** [Zettlr/Zettlr#4060] [Q&A]
Hey guys,

I was wondering if it's possible to perform a find and replace operation in multiple files simultaneously via Zettlr.
Since I foolishly changed the characters for starting/ending links to "(())" two years ago because of typing speed and even more foolishly decided to just stick with it, I now need to adapt the links to "[[]]", because Zettlr 3.0 [will only accept "[[]]" as link structure](https://zettlr.com/post/zettlr-30-public-beta).
So I just want to know, if that's possible in Zettlr, or if I'll have to do it via a script.

Greetings and thanks a lot!

**Comment by neosartre-nyc** [comment]
This is late. I don't know a way in Zettlr, but since the files are Markdown you can use VSCode or VSCodium. Open the folder in the Explorer. Choose the Magnify Glass/Find and Replace. It will work across every file in the folder. If you use multiple un-nested folders need to execute find/replace for each folder stack.
